Application/EdFi.Ods.Api/EdFi.Ods.Api.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
With a few modifications to the build script, adding a solution file for this is not needed to get it this project building and being packaged. In the original teamcity build we call `dotnet build` on `%project.file.sln%` but when you dig further into the parameters it is not actually a solution file and is instead `Ed-Fi-ODS/application/EdFi.Ods.Api/EdFi.Ods.Api.csproj`.

So what could be done for EdFi.Ods.Api is to not pass a value for the Solution parameter and only the projectFile parameter and then at the start of the build script check if Solution is empty, and if so, replace it with projectFile. This way then the calls to dotnet clean, build, and test would still work the way they are now and dotnet pack would remain unchanged since its using the projectFile already.

Reply:
Per a conversation we had in standup, this suggestion is no longer being done and the solution file will be kept.